src/services/transactionService.js

```diff
@@ -0,0 +1,42 @@
+import localforage from 'localforage';
+import walletService from './walletService';
+import { BASE_CURRENCY, TRANSACTIONS_HISTORY_KEY } from '../utils/constants';
+
+export const isUserBuying = (amount) => amount > 0;
```

Comment:
Isn't selling require to define positive amount also?

Reply:
Ah I see that amount can be negative because of selling... however that looks tricky to me.